packages/browserify/test/fixtures/secureBundling/run.sh

```diff
@@ -2,7 +2,7 @@
 set -e
 
 if [ "$WRITE_AUTO_POLICY" == "1" ]; then
-  ../../../../node/src/index.js build.js --projectRoot '../../../' --writeAutoPolicy
+  ../../../../node/src/cli.js build.js --projectRoot '../../../' --writeAutoPolicy
 else
-  ../../../../node/src/index.js build.js --projectRoot '../../../'
+  ../../../../node/src/cli.js build.js --projectRoot '../../../'
```

Comment:
This tells me we have a different understanding of projectRoot. 
My update resolves every path against it, but it seems originally it was the root for dependencies but not where the lavamoat/node/policy.json was to go.

@kumavis I need input on this.

Reply:
resolving build.js from CWD instead of the projectRoot is slightly confusing in CLI, but even more confusing with the programmatic interface.